short/codespaces-project-template-js/README.md

```diff
@@ -0,0 +1,105 @@
+
+Built to be a "choose your own adventure" template repository portfolio, we have a [React](https://reactjs.org/) based web application ready for you to easily customize and deploy using only your web browser. _No web development experience required._
+
+![JavaScript profile web application](./images/js-portfolio-site.gif "JavaScript profile web application")
+
+## About Codespaces
+
+Using Codespaces, you get [Visual Studio Code](https://visualstudio.microsoft.com/?WT.mc_id=academic-77460-alfredodeza) in the cloud, using a ["development container"](https://containers.dev/). Like a locally running version of [Visual Studio Code](https://visualstudio.microsoft.com/?WT.mc_id=academic-77460-alfredodeza), the cloud version also allows you to install extensions and use a terminal.
+
+You can also configure your development container to run a specific runtime and have it boot up with your favorite extensions.
+
+## What you will learn and requirements for this workshop
+
+| **Goal**              | Description                                    |
+| ----------------------------- | --------------------------------------------------------------------- |
+| **What will you learn**       | customizing Codespaces, web development, and web application deployment              |
+| **What you'll need**          | <ul><li>[GitHub account](https://github.com)</li><li>access to [Codespaces](https://github.com/features/codespaces)</li><li> optional [Azure for Students](https://azure.microsoft.com/en-us/free/students/) or [Azure](https://azure.microsoft.com/en-us/free/search/) account for deployment</li></ul> |
+| **Duration**                  | 35 minutes                                                                |
+| **Slides**                  | [slides](./slides.pptx) |
+
+<br/>
+
+## 🗃️ Link for this template and it's resources
+
+This template [JavaScript Portfolio Site with GitHub Codespaces](https://github.com/microsoft/codespaces-project-template-js) includes all the resources you **will need in order to demonstrate the full presentation to your audience**.
+
+Within this workshop, there is [3 points of customization](https://github.com/microsoft/codespaces-project-template-js#-customize-your-site-in-3-steps) included in the project template repository to be included. This will allow each attendee to customize their own site.
+
+Beyond 3 points of customization, there are [4 challenge tasks](https://github.com/microsoft/codespaces-project-template-js#-challenges). These allow you to customize this workshop to your audience and time you;d like to spend.
+
+Follow the [getting started instructions](https://github.com/microsoft/codespaces-project-template-js#-getting-started) to get your workshop attendees started with their poject launched within [Codespaces](https://github.com/features/codespaces). 
+
+<br />
```

Comment:
are these tags needed?

Reply:
They are, I am referring to specific sections in that readme here.